packages/server/src/service/pdfForm/massachusetts.ts

```diff
@@ -0,0 +1,27 @@
+import { MassachusettsInfo } from '../../common'
```

Comment:
Form looks good overall

MA has a primary on Sept 1st, 2020: https://www.sec.state.ma.us/ele/elepdf/2020-Primary-and-Election-Schedule-Legal-Sized.pdf

Can we mark for either the general or ask for a `primaryParty` (text box)?